dEdX/include/dEdxAnalyser.h

```diff
@@ -176,9 +220,11 @@ class dEdxAnalyser : public Processor {
   TH2* _FitBoth_Res[_nPart];  // resolution vs. lambda and vs. number of hits from fit results
   TH3* _NormBothHist_1GeV[_nPart];  // relative dEdx vs. lambda of track and vs. number of hits, for tracks with p>1GeV
   TH2* _FitBoth_Res_1GeV[_nPart];  // resolution vs. lambda and vs. number of hits from fit results, for tracks with p>1GeV
+  TH2* _NormLambdaFullHist[_nPart+1];  // relative dEdx vs. lambda of track, for tracks with p>1GeV
+  TH1* _FitLambdaFull_Res[_nPart+1];  // resolution vs. lambda from fit results
 
   // Vector array of all momentum, dE/dx, dE/dx error, tan lambda and number of hits values for each track
-  std::vector< std::array<double,5> > _dEdxVec[_nPart];
+  std::vector< std::array<double,6> > _dEdxVec[_nPart];
```

Comment:
```suggestion
  struct TrackValues {
    double mom;
    double dEdx;
    double dEdxErr;
    double tanLam;
    double nHits;
    double someValue; // what is parameter 6 now?
  };
  std::vector< TrackValues > _dEdxVec[_nPart];
```
Would make it easier to interpret the contents of this array without having to know the indices. At least the comment describing the contents of the array should be updated.